diamm/templates/website/city/city_detail.jinja2

```diff
@@ -3,13 +3,21 @@
 {% block body %}
     <div class="col-md-12">
         <h1>{{ content.name }}, <a href="{{ content.country.url }}">{{ content.country.name }}</a></h1>
-        <h2>Archives</h2>
+        {% if content.archives %}<h2>Archives</h2>{% endif %}
```

Comment:
This probably isn't correct. In this case, if `content.archives` is `None` it won't produce a header, but it will produce an empty `<ul>`. So you should wrap the whole thing in an `if` block, not just the header.